tests/unit/routes/enter-phone-number.test.ts

```diff
@@ -0,0 +1,15 @@
+const assert = require("assert");
+
+import request from "supertest";
+import { createApp } from "../../../src/app";
+
+describe("Unit testing the /enter-phone-number GET route", () => {
+  it("should return OK status", () => {
+    const app = createApp();
```

Comment:
this uses promises as its syntax it might be worthwhile changing it to opposite e.g 

```
request(createApp())
      .get("/enter-phone-number")
      .expect(200)
.end(function(err, res) {
        if (err) return done(err);
        return done();
      });
```

Reply:
updated